index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <style>
+        html, body {height: 100%; margin: 5px; padding: 0;}
+        table {background-color: linen; font-family: "Verdana", Times, serif;}
+        th {background-color: #4CAF50;color: white;}
+        tr:nth-child(even) {background-color: #f2f2f2}
+        p {font-family: "Verdana", Times, serif;}
+    </style>
+    <link rel="stylesheet" type="text/css" href="mapStyle.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>ArcTouch Code Challenge</title>
+    <script type="text/javascript" src="prototype.js"></script>
+    <script type="text/javascript" src="utils.js"></script>
+    <script type="text/javascript" src="map.js"></script>
+    <script type="text/javascript">
+        //<![CDATA[
+
+        function getPreviousRouteList() {
+            var street = getURLParameterByName("street");
+            if (street != null) {
+                document.getElementById("txtStreetName").value = street;
+                getRouteListByStreetName();
+            }
+        }
+
+        function getRouteListByStreetName() {
+            var txtStreetName = document.getElementById("txtStreetName").value.replace(/^\s+|\s+$/gm,'');
+            if (!txtStreetName || txtStreetName.trim() == "") {
+                alert("Street is required to get routes!");
+                return;
+            }
+
+            var oOptions = {
+                method: "POST",
+                postBody: "\n{\n\n\"params\": {\n\n\"stopName\": \"%" + txtStreetName + "%\"\n\n}\n\n}",

```

Comment:
The \n came from postman using the option "Generate code snippet" and gave
me the line below:
    "data": "\n{\n\n\"params\": {\n\n\"stopName\": \"%lauro
linhares%\"\n\n}\n\n}"
Since it worked I forgot to test without the \n. Code is now updated
without \n. Thanks.


2016-03-02 17:09 GMT-03:00 André Luiz Cardoso <notifications@github.com>:

> In index.html
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1#discussion_r54783079>
> :
>
> > +            if (street != null) {
> > +                document.getElementById("txtStreetName").value = street;
> > +                getRouteListByStreetName();
> > +            }
> > +        }
> > +
> > +        function getRouteListByStreetName() {
> > +            var txtStreetName = document.getElementById("txtStreetName").value.replace(/^\s+|\s+$/gm,'');
> > +            if (!txtStreetName || txtStreetName.trim() == "") {
> > +                alert("Street is required to get routes!");
> > +                return;
> > +            }
> > +
> > +            var oOptions = {
> > +                method: "POST",
> > +                postBody: "\n{\n\n\"params\": {\n\n\"stopName\": \"%" + txtStreetName + "%\"\n\n}\n\n}",
>
> Why are you using a string for the body and why are you adding all these
> \ns?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1/files#r54783079>
> .
>